### PR TITLE
Tweak the infra env empty list design

### PR DESCRIPTION
**Pull request by ammont82**
Related to issue: https://bugzilla.redhat.com/show_bug.cgi?id=2060282
Signed-off-by: Montse Ortega <mortegag@redhat.com>

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -1244,7 +1244,8 @@
     "infraEnv.bulkAction.createInfraEnv": "Create infrastructure environment",
     "infraEnv.createCluster": "Create infrastructure environment",
     "infraEnv.delete.plural": "Delete infrastructure environments",
-    "infraEnv.emptyStateHeader": "You don't have any infrastructure environments.",
+    "infraEnv.emptyStateBody": "To create a pool of hosts that is ready for cluster creation, create an infrastructure environment. Once a new infrasttructure environment is created, you can discover, provision and add hosts to it.",
```
Comment (rawagner) [inline frontend/public/locales/en/translation.json:1247]:
```suggestion
    "infraEnv.emptyStateBody": "To create a pool of hosts that is ready for cluster creation, create an infrastructure environment. Once a new infrastructure environment is created, you can discover, provision and add hosts to it.",
```

**Comment by rawagner** [comment]
@nirfarkas Im not exaclty sure if you also want to remove the New banner from top of the page ?

Screenshot from 2022-03-04 10-11-38

can you clarify ?

**Comment by nirfarkas** [comment]
@rawagner I guess we can remove it when there are infra envs already. The user already created one or more and possibly read the text in the empty state.
WDYT shirshbe@redhat.com?

**Comment by nirfarkas** [comment]
Let's remove it.

**Review by jtomasek — APPROVED** [review]
/lgtm

**Pull request merged**